helpers/validationSchemas.js

```diff
@@ -13,7 +13,7 @@ const extension = (joi) => ({
         escapeHTML: {
             validate(value, helpers) {
                 const clean = sanitizeHtml(value, {
-                    allowedTags: [`&amp`],
+                    allowedTags: [],
```

Comment:
Even though you might want to add some other package instead of this one, don't merge code that you know you're not going to use. Remove this `sanitizeHtml` and package before merging.
Whenever you find a new package you can add it in a new PR